### PR TITLE
mentions xdebug compatibility issue

### PR DESCRIPTION
**Pull request by riconeitzel**
see https://github.com/cakephp/debug_kit/issues/388

### CODE REVIEW

File: en/debug-kit.rst [hunk 1]
```diff
@@ -282,3 +282,8 @@ Helper Functions
 
 * `sql()` Dumps out the SQL from an ORM query.
 * `sqld()` Dumps out the SQL from an ORM query, and exits.
+
+Know Problems
```
Comment (bugladen) [inline en/debug-kit.rst]: Known Problems

File: en/debug-kit.rst [hunk 2]
```diff
@@ -282,3 +282,8 @@ Helper Functions
...
+=============
+
+If you work with xDebug enabled you'll end up with an exception and stack trace (https://github.com/cakephp/debug_kit/issues/388). This problem can be avoided with setting ``xdebug.collect_params`` to ``1``.
```
Comment (dereuromark) [inline en/debug-kit.rst]: technically there is also a comma after enabled.

Comment (dereuromark) [reply en/debug-kit.rst]: and "with setting" is "by setting"

Comment (garas) [reply en/debug-kit.rst]: ``` to ``0`` or ``1`` ```

0 is default, and is also OK, as stated in https://github.com/cakephp/debug_kit/issues/388#issuecomment-463139803

File: en/debug-kit.rst (hunk 1 above)
Comment (dereuromark) [reply en/debug-kit.rst]: Indeed

**Comment by saeideng** [comment]
the document moved to `cakephp/debug_kit` repo
 you can open new PR there

**Pull request closed without merging**